src/timesync/timesyncd-bus.c

```diff
@@ -43,6 +49,69 @@ static int property_get_servers(
         return sd_bus_message_close_container(reply);
 }
 
+static int method_set_runtime_servers(sd_bus_message *message, void *userdata, sd_bus_error *error) {
```

Comment:
This should refuse to set servers when the interface is managed by networkd.
See the similar bus methods in resolved.

Reply:
Hmm. At the time of calling that method the interface could not be managed by networkd yet but only later, so I don't think it makes sense to guard the setting to that? The runtime logic will give link-level servers precedence over runtime servers however - isn't that enough?

Reply:
these servers are global, not per interface. if these were per-interface I think we should refuse them if networkd supplies them, but this doesn't apply here.